wrappers/java/src/test/java/org/hyperledger/indy/sdk/wallet/ImportWalletTest.java

```diff
@@ -0,0 +1,44 @@
+package org.hyperledger.indy.sdk.wallet;
+
+import org.hyperledger.indy.sdk.IOException;
+import org.hyperledger.indy.sdk.IndyIntegrationTestWithSingleWallet;
+import org.hyperledger.indy.sdk.did.Did;
+import org.junit.Test;
+
+import java.util.concurrent.ExecutionException;
+
+import static org.hamcrest.CoreMatchers.isA;
+import static org.junit.Assert.assertEquals;
+
+
+public class ImportWalletTest extends IndyIntegrationTestWithSingleWallet {
+
+	@Test
+	public void testImportWalletWorks() throws Exception {
+		String did = Did.createAndStoreMyDid(wallet, "{}").get().getDid();
```

Comment:
maybe to add different types of records, not only dids, so we can make sure export/import works for diff record types

Reply:
Wrappers test tests only wrappers, not libindy. You can check your tests approach in https://github.com/hyperledger/indy-sdk/blob/master/doc/test-design.md